.buildkite/pipeline.yml

```diff
@@ -111,35 +111,17 @@ steps:
           - github_commit_status:
               context: "Reader Unit Tests"
       - label: "🔬 Keystone Unit Tests"
-        command: |
-          if .buildkite/commands/should-skip-job.sh --job-type validation; then
-            exit 0
-          fi
-
-          .buildkite/commands/shared-set-up.sh
-          xcodebuild \
-            -scheme Keystone \
-            -destination 'platform=iOS Simulator,OS=18.2,name=iPhone 16' \
-            test \
-            | xcbeautify
+        command: .buildkite/commands/run-unit-tests-for-scheme.sh Keystone
+        # Disabled till https://github.com/wordpress-mobile/WordPress-iOS/pull/24537 is completed
+        if: false
```

Comment:
Mmm... this syntax didn't achieve the result I hoped

<img width="768" alt="image" src="https://github.com/user-attachments/assets/a75c121b-b288-43b1-a382-8ab15946a30c" />